pipeline-scripts/buildlib.groovy

```diff
@@ -1254,4 +1275,38 @@ def assertBuildPermitted(doozerOpts) {
     }
 }
 
+/**
+ * Run elliott find-builds and attach to given advisory, if not running a DRY_RUN.
+ * It looks for builds twice (rhel-7 and rhel-8) for OCP 4.y
+ * Side-effect: Advisory states are changed to "NEW FILES" in order to attach builds.
+ *
+ * @param String[] kinds: List of build kinds you want to find (e.g. ["rpm", "image"])
+ */
+def attachBuildsToAdvisory(String[] kinds) {
+    if (params.DRY_RUN) {
```

Comment:
Since this function is in buildlib, I would prefer to pass those values via arguments instead of parameters.

Reply:
i've opted for not adding another argument to this function, instead only invoke it from the job if not DRY_RUN.